src/interface/sparse_pack.hpp

```diff
@@ -247,6 +247,18 @@ class SparsePack : public SparsePackBase {
     return bounds_h_(1, b, vidx);
   }
 
+  template <class... Args>
+  KOKKOS_INLINE_FUNCTION bool Contains(const int b, Args... args) const {
+    static_assert(sizeof...(Args) > 0, "Must pass in a variable to check existence");
+    return GetUpperBound(b, std::forward<Args>(args)...) >= 0;
+  }
+
+  template <class... Args>
+  KOKKOS_INLINE_FUNCTION bool ContainsHost(const int b, Args... args) const {
+    static_assert(sizeof...(Args) > 0, "Must pass in a variable to check existence");
+    return GetUpperBoundHost(b, std::forward<Args>(args)...) >= 0;
+  }
+
```

Comment:
Maybe add overloads so one can write 
```c++
pack.Contains<v1, v2>(b);
```
That seems cleaner to me.

Reply:
Are you imagining that returning true only if v1 and v2 are contained?

Reply:
yes, isn't that how it works currently?

Reply:
oh, nevermind. I was thinking the return was equivalent to
```c++
return (GetUpperBoundHost(b, std::forward<Args>(args)) && ...);
```

Reply:
I can make it work that way :grin:

Reply:
It works that way now. This now can return the existence status of multiple vars at once via the pack. Either with `Contains<v1, v3, v5>(b)` or with `Contains(b, v1(), v3(), v5())`